pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -156,7 +157,11 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 
 	// provide any config overrides
 	cfg.NodeName = options.NodeName
-	cfg.StreamingConnectionIdleTimeout = 5 * time.Minute // TODO: should be set
+	if server.StreamingConnectionIdleTimeout == 0 {
+		cfg.StreamingConnectionIdleTimeout = 5 * time.Minute
```

Comment:
Yes,  I have put it because the https://github.com/kubernetes/kubernetes/pull/17033 is not merged yet and would also require updating Godeps in origin, Isn't it?